sdk/servicebus/azure-servicebus/azure/servicebus/_common/message.py

```diff
@@ -516,6 +545,8 @@ def _from_list(self, messages):
             if not isinstance(each, Message):
                 raise ValueError("Only Message or an iterable object containing Message objects are accepted."
                                  "Received instead: {}".format(each.__class__.__name__))
+            if isinstance(each, (PeekMessage, ReceivedMessageBase)):
```

Comment:
I'm honestly a little bit surprised we're doing this here and not in send.  Does this have implications on the batch size calculation or otherwise that would prevent us from doing the conversion in only one place?

Reply:
implementation updated, conversion done in the send.